clients/socket-client.rst

```diff
@@ -65,8 +65,14 @@ use case would be::
 
 .. warning::
 
-    This client assume that the request is compliant with HTTP 2.0, 1.1 or 1.0 standard. So a request without a ``Host`` header, or
-    with a body but without a ``Content-Length`` will certainly fail. You can make sure that requests used by this client are valid, by
-    decorating it with :doc:`plugins </plugins/introduction>` like ``ContentLengthPlugin`` or ``HeaderDefaultsPlugin``.
+    This client assumes that the request is compliant with HTTP 2.0, 1.1 or 1.0 standard. So a request without a ``Host`` header, or
+    with a body but without a ``Content-Length`` will certainly fail.
+    If you don't trust incoming requests, we heavily recommend you to use the ``PluginClient`` with the following plugins:
```

Comment:
incoming requests sounds strange for a client. i would say: "To make sure all requests will be sent out correctly, we recommend to use ...